client/control/tests/reducers/test_columnReducer.js

```diff
@@ -0,0 +1,129 @@
+import appReducer from 'control/reducers';
+import * as actions from 'control/actions/ColumnActions';
+
+import {
+  initialState,
+} from 'control/tests/fixtures';
+
+/**
+ * Utility function to clone the initialState column array and its values
+ * (This prevents nested objects from being cloned by reference)
+ *
+ * @return {Array} New array with cloned initialState values
+ */
+const cloneInitialArray = () => JSON.parse(JSON.stringify(initialState.columns));
+
+
+/**
+ * Utility function to get the index of a column
+ * which is NOT shown by default.
+ *
+ * @return {Number} Index of non-default column to test against.
+ */
+const getNondefaultColumn = () => {
+  let foundIndex;
+  initialState.columns
+    .find((col, index) => {
+      foundIndex = index;
+      return !col.enabled;
+    });
+
+  if (!foundIndex) {
+    console.warn('Warning! No non-default columns to test against.');
+  }
+  return foundIndex;
+};
+
+/**
+ * Column reducer tests
+ */
+describe('Column reducer', () => {
+  it('should return initial state by default', () => {
+    expect(appReducer(undefined, {})).toEqual(initialState);
+  });
+
+  it('should have default columns to display', () => {
```

Comment:
My thinking was that it would prevent a case where the columns are broken or have no defaults, at which point the table wouldn't render at all. I'll remove the test, though.